xpdacq/tests/test_tools.py

```diff
@@ -0,0 +1,19 @@
+from xpdacq.tools import clean_dict
+
+def test_clean_dict():
```

Comment:
Can you set this up as a parameterized test? http://doc.pytest.org/en/latest/parametrize.html